src/helpers/client.js

```diff
@@ -0,0 +1,10 @@
+import axios from "axios";
+
+const client = axios.create({
+  baseURL: "http://127.0.0.1:8000/",
```

Comment:
For now this is ok, but consider adding support for different environments outside your code using Envorinment variables.